external/kafka/src/test/scala/org/apache/spark/streaming/kafka/KafkaStreamSuite.scala

```diff
@@ -130,7 +130,7 @@ abstract class KafkaStreamSuiteBase extends FunSuite with Eventually with Loggin
   }
 
   def createTopic(topic: String) {
-    CreateTopicCommand.createTopic(zkClient, topic, 1, 1, "0")
```

Comment:
Yeah, they just shuffled some of their classes around.  You'll notice the PR floating around for writing to kafka does the same thing, because it also upgrades kafka